### ISSUE TITLE
FB_03 - FN005:Step9: what are the specific values of the scope string that must exist?

### ISSUE THREAD

**Issue by jateeter** [energyos/OpenESPI-GreenButtonCMDTest#95]
What scope string FB, BK, and SFTP values are valid (and required) for scope string validation?

Is it required that all DataCustodians that support /Batch/Subscription/{subscriptionId} in a specific Scope?

**Comment by MartyBurns** [comment]
In the test, the gbcmdcert_target.conf has a parameter certDataScopeFBs="FB=1_4_5_15". This indicates the data FBs only that the certification is for. During FN005, the testee is requested to register enough customer accounts (usually one) so that all of these FBs get verified. For each subscription authorized, the Authorization.scope is analyzed for the subset of data FBs that should be in the data. As each FB data is successfully verified, the pending data FBs to be tested is reduced. Once all FBs have been tested, the test is completed successfully.

**Comment by dfcoffin** [comment]
I agree the FND005 test completes once all FB contained within the certDataScopeFBS variable have been processed.  However, if the SCOPE= parameter provided during the authorization request contains other FB values, they are required to be present in the customer supplied data during the data validation step.  If data is missing or invalid for any of the additional FB values then FND005 completes, but with a "FAILED" status, even if all the FBs contained in the certDataScopeFBs variable successfully passed the data validation step.
